gammapy/irf/effective_area_table.py

```diff
@@ -111,7 +111,7 @@ def __init__(self, energy_lo, energy_hi, effective_area,
         self.energy_thresh_lo = energy_thresh_lo.to('TeV')
         self.energy_thresh_hi = energy_thresh_hi.to('TeV')
 
-    def to_fits(self, header=None, **kwargs):
+    def to_fits(self, header=None, units=['TeV','m2'], **kwargs):
```

Comment:
You should never use mutable objects like lists as default arguments in Python as explained [here](https://youtu.be/sH4XF6pKKmk?t=11m).
Also this `units` option is currently not listed in `Parameters` in the docstring.

I haven't thought about this and don't have a good concept for default units here and in other places in Gammapy.
There's many options to handle this, e.g.
- Separate keyword arguments
- No option to change units at this point

Do what you think is best and it's good enough for now.
Other than lists as default keyword arguments, that is :-)